src/Compilers/CSharp/Portable/Symbols/TupleTypeSymbol.cs

```diff
@@ -23,6 +19,9 @@ internal sealed class TupleTypeSymbol : NamedTypeSymbol, ITupleTypeSymbol
         private readonly NamedTypeSymbol _underlyingType;
         private readonly ImmutableArray<TupleFieldSymbol> _fields;
 
+        internal const int chunk7 = 7; // Used for zero-based computation (array indices and such)
```

Comment:
It doesn't look like the name follows the esteblished style. I think internal static field should start with capital letter.